spec/models/aaib_report_spec.rb

```diff
@@ -0,0 +1,182 @@
+        "body" => "## Header" + ("\r\n\r\nThis is the long body of an example AAIB Report" * 10),
+        "metadata" => {
+          "date_of_occurrence" => "2015-10-10",
+          "document_type" => "aaib_report"
+        },
+        "change_history" => [],
+      },
+      "routes" => [
+        {
+          "path" => "/aaib-reports/example-aaib-report-#{n}",
+          "type" => "exact",
+        }
+      ],
+      "redirects" => [],
+      "update_type" => "major",
+    }
+  end
+
+  let(:non_aaib_report_content_item) {
+    {
+      "content_id" => SecureRandom.uuid,
+      "format" => "specialist_document",
+      "details" => {
+        "metadata" => {
+          "document_type" => "not_an_aaib_report",
+        },
+      },
+    }
+  }
+
+  let(:aaib_org_content_item) {
+    {
+      "base_path" => "/government/organisations/air-accidents-investigation-branch",
+      "content_id" => "38eb5d8f-2d89-480c-8655-e2e7ac23f8f4",
+      "title" => "Air Accidents Investigation Branch",
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-08-20T10:26:56.082Z",
+      "public_updated_at" => "2015-04-15T10:04:28.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "OT248",
+      "links" => {
+        "available_translations" => [
+          {
+            "content_id" => "38eb5d8f-2d89-480c-8655-e2e7ac23f8f4",
+            "title" => "Air Accidents Investigation Branch",
+            "base_path" => "/government/organisations/air-accidents-investigation-branch",
+            "description" => nil,
+            "api_url" => "https://www.gov.uk/api/content/government/organisations/air-accidents-investigation-branch",
+            "web_url" => "https://www.gov.uk/government/organisations/air-accidents-investigation-branch",
+            "locale" => "en"
+          }
+        ]
+      },
+      "description" => nil,
+      "details" => {}
+    }
+  }
+
+  let(:indexable_attributes) {
+    {
+      "title" => "Example AAIB Report 0",
+      "description" => "This is the summary of example AAIB Report 0",
+      "link" => "/aaib-reports/example-aaib-report-0",
+      "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example AAIB Report" * 10),
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "date_of_occurrence" => "2015-10-10",
+      "organisations" => ["air-accidents-investigation-branch"],
+    }
+  }
+
+  before do
+    @fields = [
+      :base_path,
+      :content_id,
+      :title,
+      :public_updated_at,
+      :details,
+      :description,
+    ]
+
+    @aaib_reports = []
+
+    10.times do |n|
+      @aaib_reports << aaib_report_content_item(n)
+    end
+
+    publishing_api_has_fields_for_format('specialist_document', @aaib_reports, @fields)
+
+    @aaib_reports.each do |aaib_report|
+      publishing_api_has_item(aaib_report)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
```

Comment:
You should have a corresponding `Timecop.return` in an `after` block in here (or in a global after block for every spec).